docs/en/source/opcua-IoTIntegration.rst

```diff
@@ -0,0 +1,627 @@
+
+In our case the OPC UA Agent acts as bridge between the OPC UA server and the Scorpio Context Broker behaving as an OPC UA client.
+
+Actors
+****************************************************
+
+The actors involved in the scenario are:
+
+-   **OPC UA Server**, representing the data source
+-   **OPC UA Agent**, the connector to join industrial environment to FIWARE
+-   **Scorpio Context Broker**, the broker as entry point of FIWARE platform
+
+**OPC UA Server**
+
+For tutorial purposes, you will use a simple OPC UA server (source code : https://github.com/Engineering-Research-and-Development/opc-ua-car-server).
+
+.. figure:: figures/car_schema.png
+
+It represents a car with the following structure:
+
+-   Car (obj)
+
+    -   Speed (attr)
+
+    -   Accelerate (meth)
+
+-   Stop (meth)
+
+    -   Engine (obj)
+
+        -   Temperature (attr)
+        -   Oxygen (attr)
+
+-   Sensors
+
+    -   Any number of user-defined Boolean sensor simulating a square-wave
+
+        For each sensor it is possible to define the semi-period
+		
+**OPC UA Agent**
+
+The iot-agent container relies on the presence of the Orion Context Broker and uses a MongoDB database to hold device information such as device URLs and Keys. The container is listening on two ports:
+
+-   Port 9229 is exposed to receive OPC UA measurements over HTTP from the OPC UA server.
+-   Port 4041 is exposed purely for tutorial access - so that cUrl or Postman can make provisioning commands without being part of the same network.
+
+**Scorpio Context Broker**
+
+Scorpio Context Broker can be external, however to have a black box for testing, it will be included in docker compose in order to have a self-supporting environment. Scorpio-Broker is an NGSI-LD compliant broker so we will be testing OPC UA IoT Agent for NGSI-LD tests. 
+
+Start Up
+****************************************************
+
+Please clone the repository and create the necessary images like scorpio broker and IoT agent by running the commands as shown:
+
+.. code-block:: console
+    
+     git clone https://github.com/ScorpioBroker/ScorpioBroker.git
+	 git checkout iotagent-opcua
+     cd ScorpioBroker/iotagent-opcua/docker
+
+    docker-compose up -d
+  
+     If you want to clean up and start over again you can do this with the following command:
+    
+    docker-compose down -v
+  
+**Note:** version of docker-compose should be greater than 1.21.*.
+
+After that you can run:
+
+.. code-block:: console
+
+	docker ps
+
+to check if all the required components are running
+
+Before you start the following steps, please check if your Scorpio broker is running properly.
+
+check if the Scorpio broker is running
+
+.. code-block:: console
+
+	curl <scorpio-brokerIP>:9090/actuator/health
+
+check if the IoT Agent broker is running
```

Comment:
Also add the steps to verify the Scorpio Broker deployment

Reply:
Already added